server/lib/add_user_database.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request');
+const pgClient = require('./pg_client');
+
+const getGeoLocation = (postcode, callback) => {
+  const base = 'https://api.postcodes.io/postcodes/';
+  request(base + postcode, (err, response, body) => {
+    if (err) callback(err);
+    if (!err && response.statusCode === 200) {
+      callback(err, body.result);
+    }
+  });
+};
+
+const addUserDatabase = (connection, user, callback) => {
+  const name = user.name;
+  const email = user.email;
+  const facebookId = user.facebookId;
+  const profileImgUrl = user.profileImgUrl;
+  const postcode = user.postCode;
+  const client = pgClient(connection);
+  getGeoLocation(postcode, (err, reply) => {
```

Comment:
Error will be handled by the callback that is eventually used. Should I change it to throw error?